rhel-edge-84-virt-test.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Why do you still keep this file? All things are duplicated with what you added in Jenkinsfile

Reply:
I didn't realize that my commit will reflect in this pull request.

I run test in a vm, and I usually pushed to git and then download in vm to run test, this file is just for test.

This commit should not be in this pull request, it's just for test.

Anyway, I will close this pull request as there is too many conversations here, and create a new clean on.